### DISCUSSION TITLE
[firebase_messaging] Android Notifications not showing up, sdk: >= 2.12.0

### DISCUSSION THREAD

**Discussion by justkost** [firebase/flutterfire#7115] [Q&A]
Help please! Messages are not coming when the application is not running. When app in foreground FirebaseMessaging.onMessage work correctly.

plutform: android 10
flutter: 2.5.1
sdk: >= 2.12.0

dependencies:
firebase_core: ^1.7.0
firebase_crashlytics: ^2.2.2
firebase_messaging: ^10.0.8

**Comment by y-alkhateeb** [comment]
You should create a channel by flutter local notification plugin

**Reply by justkost** [reply]
You misunderstood me, maybe I explained it wrong somehow. Local notification work great, i only have problems with background messages.

**Comment by justkost** [comment]
I don't know what the reason was, but now everything works.
